orderapi/src/main/java/fi/hel/verkkokauppa/order/model/OrderItem.java

```diff
@@ -80,6 +80,9 @@ public class OrderItem implements OrderItemSubscriptionFields, Product {
     @Field(type = FieldType.Date, format = DateFormat.date)
     LocalDate invoicingDate;
 
+    @Field(type = FieldType.Text)
+    String invoicingStatus;
```

Comment:
Could this be done like RefundAccountingStatusEnum? Then it could only have approved values.
https://github.com/City-of-Helsinki/verkkokauppa-core/blob/45911fc7653e9feecb40e3c912e3bae2b1e28c8c/orderapi/src/main/java/fi/hel/verkkokauppa/order/model/refund/Refund.java#L69